docs/04.guides/04.cookbooks/24.Query-Handling/page.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Query Handling In Lucee
+id: query-handling-lucee
+---
+
+Here we can find how the lucee supports for SQL queries. 
+
+## Query tags ##
+
+```<cfquery>``` different ways to use the tags in lucee & how we can the passed the value into the query
+
+```lucee
+<cfquery name="qry" datasource="test">
+	select * from Foo1890
+</cfquery>
+<cfdump var="#qry#" expand="false">
+```
+
+Above example just retrieve data from database.
+
+
+### Using QueryParam ###
+
+Queryparam used inside the query tag, It used to bind the value with sql statement
+
+```lucee
+<cfquery name="qry" datasource="test">
+	select * from Foo1890
+	where title=<cfqueryparam sqltype="varchar" value="Susi">
+</cfquery>
+<cfdump var="#qry#" expand="false">
+```
+
+Passing value with QueryParam, has two advantage,
```

Comment:
***toChange*** 
***Note this change includes the three lines below this comment (lines 35 through 37)***
***Please check the second advantage as it originally said "as long as the value changed" but I think it should be unchanged?***
Passing vaules with QueryParam has two advantages:
- The value you pass in QueryParam is very secure
- Lucee is able to cache the query statement and reuse it as long as the value is unchanged.